src/config/environments/dev.ts

```diff
@@ -4,6 +4,7 @@ import { Config } from './config.model';
 export const config: Config = {
     db: {
         connectionString: 'mongodb://admin:Aa123456@ds016128.mlab.com:16128/msw-dev',
+        // connectionString: 'mongodb://localhost',
```

Comment:
lets create local.ts file with this line

Reply:
ok